paymentapi/src/main/java/fi/hel/verkkokauppa/payment/model/refund/RefundPayment.java

```diff
@@ -0,0 +1,78 @@
+package fi.hel.verkkokauppa.payment.model.refund;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.elasticsearch.annotations.DateFormat;
+import org.springframework.data.elasticsearch.annotations.Document;
+import org.springframework.data.elasticsearch.annotations.Field;
+import org.springframework.data.elasticsearch.annotations.FieldType;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+@Document(indexName = "refund_payments")
+public class RefundPayment {
+
+	// refundId (Refund model id) + timestamp (Like creating paymentId)
+	@Id
+	String refundPaymentId;
```

Comment:
I think these fields should also be private. Most of the Document model classes that we have does not leverage private field decleration, but I think we should use them and especially for the new classes. It only adds a safeguard to the fields accessibility scope.